test/evp_test.c

```diff
@@ -1225,12 +1225,14 @@ static int mac_test_run_mac(EVP_TEST *t)
             goto err;
         }
         tmpval = strchr(tmpkey, ':');
-        if (tmpval != NULL)
+        if (tmpval != NULL) {
```

Comment:
There is something very funky about the indenting here - where is the matching } for this one??

Reply:
Oops, I'll just remove that